msgraph/models.go

```diff
@@ -693,12 +693,66 @@ type ConditionalAccessSessionControls struct {
 }
 
 type ConditionalAccessUsers struct {
-	IncludeUsers  *[]string `json:"includeUsers,omitempty"`
-	ExcludeUsers  *[]string `json:"excludeUsers,omitempty"`
-	IncludeGroups *[]string `json:"includeGroups,omitempty"`
-	ExcludeGroups *[]string `json:"excludeGroups,omitempty"`
-	IncludeRoles  *[]string `json:"includeRoles,omitempty"`
-	ExcludeRoles  *[]string `json:"excludeRoles,omitempty"`
+	IncludeUsers                 *[]string                               `json:"includeUsers,omitempty"`
+	ExcludeUsers                 *[]string                               `json:"excludeUsers,omitempty"`
+	IncludeGroups                *[]string                               `json:"includeGroups,omitempty"`
+	ExcludeGroups                *[]string                               `json:"excludeGroups,omitempty"`
+	IncludeRoles                 *[]string                               `json:"includeRoles,omitempty"`
+	ExcludeRoles                 *[]string                               `json:"excludeRoles,omitempty"`
+	IncludeGuestsOrExternalUsers *ConditionalAccessGuestsOrExternalUsers `json:"includeGuestsOrExternalUsers"`
+	ExcludeGuestsOrExternalUsers *ConditionalAccessGuestsOrExternalUsers `json:"excludeGuestsOrExternalUsers"`
+}
+
+type ConditionalAccessGuestsOrExternalUsers struct {
+	GuestOrExternalUserTypes *[]ConditionalAccessGuestOrExternalUserType `json:"guestOrExternalUserTypes,omitempty"`
+	ExternalTenants          *ConditionalAccessExternalTenants           `json:"externalTenants,omitempty"`
+}
+
+type ConditionalAccessExternalTenants struct {
+	MembershipKind *string   `json:"membershipKind,omitempty"`
+	Members        *[]string `json:"members,omitempty"`
```

Comment:
Missed the type alias

```suggestion
	MembershipKind *ConditionalAccessExternalTenantsMembershipKind `json:"membershipKind,omitempty"`
	Members        *[]string                                       `json:"members,omitempty"`

```